packages/canner/src/action/actionToMutation.js

```diff
@@ -113,17 +113,17 @@ export default function actionToMutation(action: Action<ActionType>) {
 
 function genCreateInputType(action) {
   const key = action.payload.key;
-  return `${transformKey(key)}CreateInput!`
+  return `${transformKey(key.toLowerCase())}CreateInput!`
 }
 
 function genUpdateInputType(action) {
   const key = action.payload.key;
-  return `${transformKey(key)}UpdateInput!`;
+  return `${transformKey(key.toLowerCase())}UpdateInput!`;
 }
 
 function genWhereInputType(action) {
   const key = action.payload.key;
-  return `${transformKey(key)}WhereUniqueInput!`;
+  return `${transformKey(key.toLowerCase())}WhereUniqueInput!`;
 }
 
 export function transformKey(key: string) {
```

Comment:
We don't have to add `.toLowerCase()` to every key, just use the [capitalize](https://lodash.com/docs/4.17.11#capitalize) in `transformKey`
```js
export function transformKey(key: string) {
  return capitalize(pluralize.singular(key));
}
```